app/src/main/res/raw/menu.json

```diff
@@ -387,6 +387,94 @@
 				],
 				"icon": "https://www.opensouthcode.org/logo.png"
 			}
+		},
+		{
+			"version": 2017230500,
+			"url": "https://raw.githubusercontent.com/heysadboy/Giggity-Parser/master/Parser/sample.xml",
+			"title": "FOSSASIA 2016",
+			"start": "2016-08-10",
+			"end": "2016-08-12",
+			"metadata": {
+				"links": [
+					{
+						"url": "http://fossasia.org",
+						"title": "Website"
+					}
+				],
+              "icon": "https://raw.githubusercontent.com/fossasia/open-event-android/development/android/app/src/main/res/drawable/splash_logo.png",
```

Comment:
The icon needs to be square, monochrome and transparent. I think it does cover the last 2 but it's not properly square. :-( Would you have another one, maybe what you use as favicon?